config/locales/ar.yml

```diff
@@ -3587,7 +3587,6 @@ ar:
     card_code: "رمز البطاقة"
     card_number: "رقم البطاقة"
     category: "الفئة"
-    created_successfully: "تم إنشاؤه بنجاح"
```

Comment:
We usually don't change locales apart from the source `en.yml` in pull requests. The reason is that all changes will get overridden when we create a new release. Transifex pull the source and compiles all the other locales for us. We then just download them and add them to the master branch during release creation. So Transifex is the source of truth.

That said, I don't see any harm here. You don't need to modify the commits. Unless we create a release first and then you may get a merge conflict.

Reply:
Interesting! That was my initial hunch, but then I had the question: if the source of truth is the `en.yml`, shouldn't Transfirex make sure all other yml files have the same schema? For example, before this PR, `en.yml` had two extra keys that some other files did not have. I would have expected Transfirex to have added those keys to all files automatically.

Reply:
> But when are new translation keys in config/locales/en.yml pushed to Transfirex so that they can be translated by translators?

We configured transifex to pull en.yml from Github. I think they do that every 12 hours or so. But translators don't always translate all keys.